__tests__/main.test.ts

```diff
@@ -11,6 +11,7 @@ import {
 import { randomUUID } from "crypto";
 
 const INVALID_URL = "INVALID_URL";
+const INVALID_REGION = "INVALID_REGION";
 const TEST_ACCESS_TOKEN = randomUUID().toString();
 const TEST_SECRETS = [`\t${randomUUID().toString()} > TEST_VALUE`];
 const DEFAULT_BASE_URL = "";
```

Comment:
We need a `DEFAULT_CLOUD_REGION` to be set to whatever is in the `action.yml` as default (us).
And then passed on in every single test (cloud_region) to simulate the default optional behaviour.

Also since `DEFAULT_IDENTITY_URL` and `DEFAULT_API_URL` are now empty string in `action.yml` as defaults, we should set them to empty string too in the test. (Although those are used as the test expectancy url's, so this will need to be adjusted accordingly to not use those variables)